### PR TITLE
br: no domain to run log command

### LINKED ISSUE #52088: no need to start domain for br log operation
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
1. `./br log start/stop/resume...`

### 2. What did you expect to see? (Required)
1. less memory used
### 3. What did you see instead (Required)
2. oom
### 4. What is your TiDB version? (Required)
v6.5 v7.1 v7.5 master

### PR DESCRIPTION
**Pull request by Leavrth**
### What problem does this PR solve?

Issue Number: close #52088 

Problem Summary:
no need to start domain for br log operation
### What changed and how does it work?
don't run domain when use log start/stop... command.
### Check List

Tests 

- [x] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by 3pointer — APPROVED** [review]

File: br/pkg/task/stream.go [hunk 1]
```diff
@@ -454,13 +454,8 @@ func (s *streamMgr) backupFullSchemas(ctx context.Context, g glue.Glue) error {
 	return nil
 }
 
-func (s *streamMgr) checkStreamStartEnable(g glue.Glue) error {
-	se, err := g.CreateSession(s.mgr.GetStorage())
```
Comment (Leavrth) [inline br/pkg/task/stream.go:458]: The function `CreateSession` doesn't call `session.InitMDLVariable`. If it skips creating domain in mgr, creating index would be stuck in state `write only` or `delete only`.

**Review by YuJuncen — APPROVED** [review]

**Pull request merged**